frontend/src/js/common/helpers/dateHelper.ts

```diff
@@ -213,3 +214,16 @@ export function getFirstAndLastDateOfRange(dateRangeStr: string): {
 
   return { first, last };
 }
+
+export function getMonthName(date: Date): string {
+  const locale = i18next.language === "de" ? de : enGB;
```

Comment:
Could you reuse `useDateLocale` from some lines above (+ make these functions hooks)? That way, once we allow switching language, it will still work.